msr-flows/flow-tests/input.py

```diff
@@ -0,0 +1,135 @@
+# Imports
+from time import time, sleep
+import serpent_builder as sb
+import complex_serpent_builder as csb
+import restart_serpent_builder as rsb
+import os
+import res_dep_analysis as rda
+import subprocess
+import os.path
+from os import path
+
+# User Definitions
+INPUT_NAME = 'cycle_test'
+DIR_NAME = 'msr_cycle_test'
+NUM_CYCLES = 3
+CYCLE_TIME_SECONDS = 2
+CYCLE_STEP_SIZE_SECONDS = 1
+OUTPUT_NAME = 'output'
+PLOTTING = True
+NON_CYCLE = False
+MULTI_CORE = False
+RESTART_CYCLE = True
+
+# Functions
+
+
+def run_script(INPUT_NAME, OUTPUT_NAME, input_script):
+    '''
+    Writes the input_script string to a file with the INPUT_NAME
+    which is run and outputs to the OUTPUT_NAME.
+    '''
+    with open(INPUT_NAME, 'w+') as input_file:
+        input_file.write(input_script)
+    os.system('./sss2 -omp 32 ' + str(INPUT_NAME) + ' > ' + str(OUTPUT_NAME))
+    return
+
+
+def check_wrk_file(INP_NAME):
+    '''
+    Allows the script to continue once the .wrk file is generated
```

Comment:
every function should have the following:
```
Parameters:
Returns:
```
Those should define what every argument of the function is for.

Reply:
More details here: https://numpydoc.readthedocs.io/en/latest/format.html